core/backend/src/IModelDb.ts

```diff
@@ -2651,9 +2652,8 @@ export class SnapshotDb extends IModelDb {
  * - may be opened without supplying any user credentials
  * - may be opened read/write
  * - may optionally support undo/redo via [[TxmManager]]
- * - cannot apply a changeset to nor generate a changesets
- * - are only available to authorized applications
- * @internal
+ * - cannot apply a changeset to nor generate a changesets (since there is no timeline from which to get/push changesets)
+ * @public

```

Comment:
Can you add something like `@see [BriefcaseConnection.openStandalone]($frontend) to open a StandaloneDb on the frontend`? The relationship between StandaloneDb and BriefcaseConnection is not obvious.

Reply:
However `openStandalone` is currently `internal`.

Reply:
made it @public